handlers/passport.js

```diff
@@ -0,0 +1,63 @@
+const passport = require('passport')
+const bcrypt = require('bcrypt-nodejs')
+const JwtStrategy = require('passport-jwt').Strategy
+const ExtractJwt = require('passport-jwt').ExtractJwt
+const LocalStrategy = require('passport-local')
+const db = require('../db')
+
+// Create local strategy
+const localOptions = { usernameField: 'email' }
+const localLogin = new LocalStrategy(localOptions, async function(
+  email,
+  password,
+  done
+) {
+  // Verify this email and password, call done with the user
```

Comment:
I wouldn't recommend writing too many comments, because code is pretty much self explanatory and it doesn't need to be commented. Use comments only for snippets which are hard to understand. This is something like recommendation for future PR. You can read more about that at: https://johnsy.com/blog/2012/10/31/comments-are-lies/

Reply:
Thanks for tips. I'll check it out!